INSTALL.txt

```diff
@@ -52,48 +55,88 @@ Homebrew
 .. code-block:: console
 
     $ brew install graphviz
-    $ pip install --pre pygraphviz
+    $ pip install pygraphviz
 
 Advanced
 ========
 
 The two main difficulties are
-(1) informing pip where Graphviz is installed and
-(2) installing Graphviz.
-
+(1) installing Graphviz and
+(2) informing pip where Graphviz is installed.
```

Comment:
```suggestion
   1. installing Graphviz and
   2. informing pip where Graphviz is installed.
```
This should result in these points being rendered as an enumerated list (as of now they are just inline text).